studio/schemas/deskStructure.ts

```diff
@@ -64,21 +64,21 @@ export default (S: StructureBuilder) =>
                   S.document()
                     .schemaType(blogId)
                     .documentId(blogId)
-                    .title("Blog Overview & Settings")
+                    .title("Blog Overview & Settings"),
                 ),
               S.listItem()
                 .title("Posts")
                 .icon(ComposeIcon)
                 .child(S.documentTypeList(postId).title("Posts")),
-            ])
+            ]),
         ),
       S.listItem()
         .title("Salary and Benefits")
         .icon(HeartIcon)
```

Comment:
[this is not an actual PR comment] would it be nice with a ConfettiIcon instead of a heart? We could also ask design, this is as much part of the product as the actual website. 🤔  Perhaps we should let them in on this later on.

Reply:
We can invite the designers and let them make decisions here!